### PR TITLE
Cu 86bzppqyu refont accueil page

### PR DESCRIPTION
**Pull request by MarvinFlm**
# Description

Refont of the accueil page to fit with the current UI

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to a clickup task
- [x] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

### CODE REVIEW

File: patient_mobile/lib/screens/dashboard/accueil_page.dart [hunk 1]
```diff
@@ -48,142 +80,111 @@ class _HomePageState extends State<HomePage> {
...
+                height: 4,
+              ),
+              const SizedBox(height: 8),
+              nextAppointment != null
+                  ? AppoitementCard(
+                      startDate: DateTime.fromMillisecondsSinceEpoch(
+                          nextAppointment['start_date'] * 1000),
+                      endDate: DateTime.fromMillisecondsSinceEpoch(
+                          nextAppointment['end_date'] * 1000),
+                      doctor: "Dr. Jean Dupont",
```
Comment (lucas-louis) [inline patient_mobile/lib/screens/dashboard/accueil_page.dart:138]: Really ??

File: patient_mobile/lib/screens/dashboard/accueil_page.dart [hunk 2]
```diff
@@ -48,142 +80,111 @@ class _HomePageState extends State<HomePage> {
...
+                      onTap: () {},
```
Comment (lucas-louis) [inline patient_mobile/lib/screens/dashboard/accueil_page.dart:139]: You need to open a modal when the card is clicked

**Review by lucas-louis — CHANGES_REQUESTED** [review]
- The green circle needs to be replace by the avatar of the profile
- The divider needs to be thicker
- Not enough spacing between the "Besoin d'un nouveau rdv" et the button
- Not enough spacing between the appointment and the new appointment section

Treatments:
- Bad border radius
- Bad period name "Après-midi" => "Midi"

**Review by lucas-louis — CHANGES_REQUESTED** [review]
Can't load the info of the patient: got "Erreur lors du chargement des données"

**Review by lucas-louis — CHANGES_REQUESTED** [review]
- The divider needs to be thicker
- Not enough spacing between the appointment and the new appointment section
- You need to display the firstname not the name of the patient

Treatments:
- Bad border radius
- The name of the period is not good (too small?)
- Bad fontWeight for the medicines
- The checkbox needs to be on the left (not the right)
- The name of the medicine needs to be crossed when the checkbox is checked

**Review by lucas-louis — CHANGES_REQUESTED** [review]
Treatments;
- The name of the period is not good (too small and too thick)
- The text is not next to the checkbox
- The name of the treatment is too bolded

**Review by lucas-louis — APPROVED** [review]
LGTM

**Review by SpiritGY — APPROVED** [review]

**Pull request merged**